src/main/webapp/css/main.css

```diff
@@ -24,7 +24,7 @@ body {
 }
 
 nav {
-  background-color: #283593;
+  background-color: #037596;
 }
 
 nav a {
```

Comment:
If you'd like you could try playing around with using padding instead of margin on these links. That might give you more of a tabbed look in the hover effect, if that's something you're interested in. Doesn't have to be today if you don't want to mess around with the layout too much!

Reply:
I'll check it out!!